src/main/java/uk/gov/register/views/representations/turtle/EntryTurtleWriter.java

```diff
@@ -36,8 +36,9 @@ protected Model rdfModelFor(Entry entry, boolean includeKey) {
         String entryNumber = Integer.toString(entry.getEntryNumber());
         Resource resource = model.createResource(entryUri(entryNumber).toString())
                 .addProperty(entryNumberProperty, entryNumber)
-                .addProperty(entryTimestampProperty, entry.getTimestampAsISOFormat())
-                .addProperty(itemProperty, model.createResource(itemUri(entry.getSha256hex().encode()).toString()));
+                .addProperty(entryTimestampProperty, entry.getTimestampAsISOFormat());
+
+        entry.getItemHashes().forEach(hash -> resource.addProperty(itemProperty, model.createResource(itemUri(hash.encode()).toString())));
```

Comment:
There's a bug here where if an item is included in an entry more than once, it is only written once to the ttl output.

For example:

RSF:
```assert-root-hash	sha-256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
add-item	{"address":"100101030506","end-date":"","name":"Glanaman Home Tution Centre","school":"402175","start-date":"2007-11-07"}
add-item	{"address":"10034660829","end-date":"","name":"First Steps","school":"402161","start-date":"2007-11-06"}
add-item	{"address":"100101037877","end-date":"2010-08-31","name":"Options Education Programme","school":"402160","start-date":"2007-11-06"}
append-entry	2016-11-24T17:20:33Z	sha-256:4e5ece9111ce1024467b4094ac3f477f83ca7ed100e332e06330d607245b0537	402175
append-entry	2016-11-24T17:20:33Z	sha-256:088bb2124883bf2b82812f6d290cf860d9699fb17368e66ee442c3875162bfa1	402161
append-entry	2016-11-24T17:20:33Z	sha-256:cbc12a2ff953c5751b34706990ff32af935942b4901254079457e6e5b5c35c04	402160
append-entry	2016-11-24T17:20:33Z	sha-256:cbc12a2ff953c5751b34706990ff32af935942b4901254079457e6e5b5c35c04;sha-256:088bb2124883bf2b82812f6d290cf860d9699fb17368e66ee442c3875162bfa1;sha-256:cbc12a2ff953c5751b34706990ff32af935942b4901254079457e6e5b5c35c04	402160
append-entry	2016-11-24T17:20:33Z	sha-256:cbc12a2ff953c5751b34706990ff32af935942b4901254079457e6e5b5c35c04;sha-256:088bb2124883bf2b82812f6d290cf860d9699fb17368e66ee442c3875162bfa1	402160```

Output from `/entry/4.ttl`:
```@prefix register-metadata:  <https://openregister.github.io/specification/#> .

<http://school.openregister.dev:8080/entry/4>
      register-metadata:entry-number-field
              "4" ;
      register-metadata:entry-timestamp-field
              "2016-11-24T17:20:33Z" ;
      register-metadata:item-resource
              <http://school.openregister.dev:8080/item/sha-256:088bb2124883bf2b82812f6d290cf860d9699fb17368e66ee442c3875162bfa1> , <http://school.openregister.dev:8080/item/sha-256:cbc12a2ff953c5751b34706990ff32af935942b4901254079457e6e5b5c35c04> ;
      register-metadata:key-field
              "402160" .```

Expected:
```@prefix register-metadata:  <https://openregister.github.io/specification/#> .

<http://school.openregister.dev:8080/entry/4>
      register-metadata:entry-number-field
              "4" ;
      register-metadata:entry-timestamp-field
              "2016-11-24T17:20:33Z" ;
      register-metadata:item-resource
              <http://school.openregister.dev:8080/item/sha-256:088bb2124883bf2b82812f6d290cf860d9699fb17368e66ee442c3875162bfa1> , <http://school.openregister.dev:8080/item/sha-256:cbc12a2ff953c5751b34706990ff32af935942b4901254079457e6e5b5c35c04> , <http://school.openregister.dev:8080/item/sha-256:088bb2124883bf2b82812f6d290cf860d9699fb17368e66ee442c3875162bfa1> ;
      register-metadata:key-field
              "402160" .```